packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -1,3 +1,4 @@
+import type { ErrorObject } from 'ajv';
```

Comment:
I think we can't use `ajv` here. If we do, then it'll be part of our emitted typings and as such, will break if someone will use it without `ajv`. This type is [rather easy](https://github.com/ajv-validator/ajv/blob/16af695b9e19e37fe064145fd33d7a630e9263c0/lib/types/index.ts#L86-L97), so we could inline the part of it we're using.